background/messages/mCreateWindow.ts

```diff
@@ -1,10 +1,21 @@
-import createWindowUrl from "url:~background/messages/mCreateWindow.html"
-import { windows } from "webextension-polyfill"
+import {
+  runtime,
+  tabs,
+  windows,
+  type Tabs,
+  type Windows
+} from "webextension-polyfill"
 
 import { type PlasmoMessaging } from "@plasmohq/messaging"
 
 export type RequestBody = {
-  in: string
+  method: string
+  params: [
+    {
+      origin: string
+      account: string
+    }
+  ]
```

Comment:
I feel that we could unwrap the body to only:

```suggestion
  origin: string
  account: string
```